Subject:       Use different header location for oneTBB when using tbb::split
Patch author:  Orson Teodoro <orsonteodoro@hotmail.com>
Date:          Wed Jun 30 08:43:49 PM PDT 2021 (Unix timestamp: 1625111029)

Copyright notice for nanovdb/util/Range.h:

// Copyright Contributors to the OpenVDB Project
// SPDX-License-Identifier: MPL-2.0

You must add -DNANOVDB_USE_ONETBB to CPPFLAGS.

diff -urp nanovdb.orig/nanovdb/util/Range.h nanovdb/nanovdb/util/Range.h
--- nanovdb.orig/nanovdb/util/Range.h	2020-09-24 20:19:30.000000000 -0700
+++ nanovdb/nanovdb/util/Range.h	2021-06-30 20:40:40.741878653 -0700
@@ -14,9 +14,13 @@
 #ifndef NANOVDB_RANGE_H_HAS_BEEN_INCLUDED
 #define NANOVDB_RANGE_H_HAS_BEEN_INCLUDED
 
-#ifdef NANOVDB_USE_TBB
-#include "tbb/tbb_stddef.h"// for tbb::split
-#endif
+#ifdef NANOVDB_USE_TBB						// oteodoro - added line
+  #ifdef NANOVDB_USE_ONETBB					// oteodoro - indent
+    #include "tbb/blocked_range.h"// for tbb::split		// oteodoro - added line
+  #else								// oteodoro - added line
+    #include "tbb/tbb_stddef.h"// for tbb::split		// oteodoro - indent
+  #endif							// oteodoro - indent
+#endif								// oteodoro - added line
 
 namespace nanovdb {
 
